nautobot/cloud/models.py

```diff
@@ -18,7 +22,7 @@
     "graphql",
     "webhooks",
 )
-class CloudAccount(PrimaryModel):
+class CloudAccount(CloudTypeCompatibleMixin, PrimaryModel):
```

Comment:
I think CloudType `content_types` is meant to be for CloudNetwork and CloudService only at present? CloudAccount doesn't have `extra_config` so the schema defined by a CloudType isn't really relevant.

Reply:
There is no model `CloudService` at present

Reply:
I'm aware, yes :). So in the immediate term it would be only `CloudNetwork`.